_posts/2017-11-20-owasp-ctf.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Mini-CTF OWASP
+tags: ctf
```

Comment:
We try to avoid introducing new tags after 5 years. Have a look at the archive's page "topics" (which are the tags) and apply relevant ones. Since I was there I can help you a bit, put these in there: `web sql-injection linux tutorial xss`